packages/grafana-ui/src/components/TimeSeries/utils.ts

```diff
@@ -294,21 +294,18 @@ export const preparePlotConfigBuilder: UPlotConfigPrepFn<{
         }
       }
 
-      let axisColorOpts = {};
-
-      if (axisColor) {
-        axisColorOpts = {
-          border: {
-            show: true,
-            width: 1,
-            stroke: axisColor,
-          },
-          ticks: {
-            stroke: axisColor,
-          },
-          color: customConfig.axisColorMode === AxisColorMode.Series ? axisColor : undefined,
-        };
-      }
+      let axisColorOpts = {
```

Comment:
I wonder if this naming still makes sense as this is actually more than just axis color options 🤔 

maybe it would make more sense to rename this as `const axisDisplayOptions` or something?
```suggestion
      const axisDisplayOptions = {
```

Reply:
At first, I thought that the `tweakAxis` method somehow changes `axisColorOpts` and decided not to change it to const, but now I see that it accepts a newly created object + deconstructs `axisColorOpts`, so we're safe 🙂 I don't know how I missed that 🤦🏻‍♂️, thanks for pointing it out.

And yes, I totally agree with the new name 👍🏻